### ISSUE TITLE
Users for translations

### ISSUE THREAD

**Issue by federicocalvo** [frappe/erpnext#18607]
Hello, I request the creation of two translation users.

The github usernames are @Don-Leopardo  and @fproldan.

This is what they mentioned when writing to frappe support

Regards

**Comment by kennethsequeira** [comment]
@federicocalvo @Don-Leopardo @fproldan 

You can sign up directly on the translator portal,  go to **Contributed Translation** doctype and begin verifying them.

In case you have already signed up and aren't able to view the same, post a comment here.

**Comment by federicocalvo** [comment]
Hi, i dont understand. I sign up in translate.erpnext.com ,but I can't find a section *Contributed Translation*.

**Comment by kennethsequeira** [comment]
There is some issue on the portal. Checking this now.

**Comment by sahil28297** [comment]
The issue is resolved, can you check again

**Comment by Don-Leopardo** [comment]
I have tried to create a new account from https://translate.erpnext.com/login#signup. I received 50 (exact number) emails about my new account with links to set the password. The links didn't work so I tried to use https://translate.erpnext.com/login#forgot and it sent me an email with a new link, that link didn't work neither.

**Illustrative screenshots:**
![Captura de Pantalla 2019-08-20 a la(s) 10 41 56](https://user-images.githubusercontent.com/46027152/63352866-3cf84800-c338-11e9-86e6-6f88a0ac75db.png)

![Captura de Pantalla 2019-08-20 a la(s) 10 45 22](https://user-images.githubusercontent.com/46027152/63352865-3cf84800-c338-11e9-832f-a63a6909dd94.png)
